src/XRP/common-xrpl-client.ts

```diff
@@ -0,0 +1,277 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
what's this for

Reply:
not sure but I was able to remove it with no repercussions so it's gone in both clients now.. probably a relic of long ago